WORKSPACE

```diff
@@ -113,10 +117,36 @@ go_register_toolchains(go_version = "1.15.11")
 # Configure nodeJS.
 load("@build_bazel_rules_nodejs//:index.bzl", "yarn_install")
 
+# install external dependencies for pkg/ui package
 yarn_install(
     name = "npm",
     package_json = "//pkg/ui:package.json",
     yarn_lock = "//pkg/ui:yarn.lock",
+    frozen_lockfile = False,
+)
+
+# install external dependencies for pkg/ui/src/js package (used to generate protobuf js client)
+yarn_install(
+    name = "npm_proto_js",
+    package_json = "//pkg/ui/src/js:package.json",
+    yarn_lock = "//pkg/ui/src/js:yarn.lock",
+    args = ["--modules-folder", "./node_modules"],
+    frozen_lockfile = False,
+)
+
+# install external dependencies for pkg/ui/cluster-ui
+yarn_install(
+    name = "npm_cluster_ui",
+    package_json = "//pkg/ui/cluster-ui:package.json",
+    yarn_lock = "//pkg/ui/cluster-ui:yarn.lock",
+    package_path = "pkg/ui/cluster-ui",
+    links = {
+        "@cockroachlabs/crdb-protobuf-client": "//pkg/ui/src/js:ui_protos_lib",
+    },
```

Comment:
target is linked to be able access to it with `@npm_cluster_ui//@cockroachlabs/crdb-protobuf-client`